<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RepoActorsDialog</class>
 <widget class="QDialog" name="RepoActorsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>703</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Actors available: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetMain" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Current actor:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="boxActors">
        <property name="maxVisibleItems">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCloneActor">
        <property name="text">
         <string>Clone</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonRemoveActor">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonNewActor">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Add new</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetBottom" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupLeft">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="title">
         <string>Repository and Production params</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="labelName">
           <property name="font">
            <font>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="lineName">
           <property name="placeholderText">
            <string>geralt_of_bolivia</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelPath">
           <property name="font">
            <font>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Template path:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="linePath">
           <property name="placeholderText">
            <string>dlc/kovir/data/entities/geralt.w2ent</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelDefApp">
           <property name="font">
            <font>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Default appearance:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QComboBox" name="boxDefApp"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelApps">
           <property name="font">
            <font>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Appearances:&lt;br/&gt;(one line per appearance)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QPlainTextEdit" name="textApps">
           <property name="placeholderText">
            <string>geralt_app_example</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labelTags">
           <property name="font">
            <font>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tags:&lt;br/&gt;(one line per tag)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPlainTextEdit" name="textTags">
           <property name="placeholderText">
            <string>TAG_EXAMPLE</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QCheckBox" name="checkVoicetag">
           <property name="text">
            <string>By voicetag</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupRight">
        <property name="title">
         <string>Storyboard defaults</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkPlacement">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Default placement:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="InputPlacement" name="widgetPlacement" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkPose">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Default pose:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="InputPose" name="widgetPose" native="true"/>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="buttonClonePose">
           <property name="text">
            <string>Clone pose</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkMimicPose">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Default mimic:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="InputMimicPose" name="widgetMimicPose" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InputPlacement</class>
   <extends>QWidget</extends>
   <header>InputPlacement.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InputPose</class>
   <extends>QWidget</extends>
   <header>InputPose.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InputMimicPose</class>
   <extends>QWidget</extends>
   <header>InputMimicPose.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RepoActorsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RepoActorsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
